### DISCUSSION TITLE
Documentation translations

### DISCUSSION THREAD

**Discussion by merongivian** [yippee-fun/phlex#256] [Ideas]
I know we don't have much documentation yet but would be nice to have some spanish translations, and i can work on it of course :). I can even start looking at the codebase and add more documentation that could be used in the main docs 🤔.

**Comment by joeldrapper** [comment]
That would be awesome! The documentation is really volatile at the moment. Shall we revisit this once things have settled a bit?

**Reply by merongivian** [reply]
Sure!, sounds good :)
